src/ILLink.RoslynAnalyzer/TrimAnalysis/RequireDynamicallyAccessedMembersAction.cs

```diff
@@ -7,9 +7,9 @@
 
 namespace ILLink.Shared.TrimAnalysis
 {
+#pragma warning disable CS0282 // There is no defined ordering between fields in multiple declarations of partial struct
```

Comment:
Cleaner way is to add `[StructLayout (LayoutKind.Auto)]` on the struct - it will shut down the warning and also tells the runtime it's free to optimize the layout if it helps.

Reply:
Also please add onto the shared source.